specification/communication/data-plane/Rooms/preview/2022-02-01-preview/communicationservicesrooms.json

```diff
@@ -0,0 +1,707 @@
+            "schema": {
+              "$ref": "#/definitions/RoomModel"
+            }
+          },
+          "default": {
+            "description": "Error response",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "x-ms-client-name": "ErrorCode",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Add participants": {
+            "$ref": "./examples/Rooms_AddParticipants.json"
+          }
+        }
+      }
+    },
+    "/rooms/{roomId}/participants:update": {
+      "post": {
+        "tags": [
+          "Participants"
+        ],
+        "summary": "Update participants in a room.",
+        "operationId": "Rooms_UpdateParticipants",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "roomId",
+            "description": "Id of the room.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "updateParticipantsRequest",
+            "description": "Participants in a room to be updated.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/UpdateParticipantsRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The participants were successfully updated. The action returns an updated `Room`",
+            "schema": {
+              "$ref": "#/definitions/RoomModel"
+            }
+          },
+          "default": {
+            "description": "Error response",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "x-ms-client-name": "ErrorCode",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Update participant": {
+            "$ref": "./examples/Rooms_UpdateParticipants.json"
+          }
+        }
+      }
+    },
+    "/rooms/{roomId}/participants:remove": {
+      "post": {
+        "tags": [
+          "Participants"
+        ],
+        "summary": "Remove participants from a room.",
+        "operationId": "Rooms_RemoveParticipants",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "roomId",
+            "description": "room id to remove the participants from.",
```

Comment:
Nit: format `Room` like the others? A few more instances of lowercased Room below and some that don't have the ticks.

Reply:
Updated all references to lowercase room to be consistent across the spec.